spyder/plugins/editor/widgets/editor.py

```diff
@@ -1505,6 +1505,10 @@ def close_file(self, index=None, force=False):
 
             self.add_last_closed_file(finfo.filename)
 
+            # Remove autosave file on successful close to work around #9265

```

Comment:
... around *issue* 9265 ...

Reply:
Done

Reply:
@cam-gerlach could you add this comment on hiw we reference issues somewhere on the wiki or even on the PR template ? 

Thanks! Great work

Reply:
@goanpeca I reorganized the [Code style](https://github.com/spyder-ide/spyder/wiki/Dev:-Coding-Style) page and added this along with a number of other standards and conventions